src/backend/executor/cypher_set.c

```diff
@@ -454,9 +467,13 @@ static void process_update_list(CustomScanState *node)
                                                   update_item->prop_name,
                                                   new_property_value,
                                                   remove_property);
-
+        
         resultRelInfo = create_entity_result_rel_info(
             estate, css->set_list->graph_name, label_name);
+        //relid = RelationGetRelid(resultRelInfo->ri_RelationDesc); 
```

Comment:
Don't understand these change?

Reply:
It is probably better to pull the PR down locally and look at it there. There are enough changes made that these diffs are a bit confusing.